requirements/ci.in

```diff
@@ -3,3 +3,7 @@
 -c constraints.txt
 
 tox                       # Virtualenv management for tests
+
+# add below 3 packages as mypy giving errors of missing stubs
+djangorestframework-stubs
+types-requests
```

Comment:
Shouldn't this go to the base requirements?

Reply:
Actually we don't need these in base, 
these requirements are already going to dev.txt and ci.txt through ci.in which are installed during tests.